pkgs/tools/security/gnupg/24.nix

```diff
@@ -85,7 +86,7 @@ stdenv.mkDerivation rec {
 
   enableParallelBuilding = true;
 
-  passthru.tests.connman = lib.nixosTests.gnupg;
+  passthru.tests.connman = nixosTests.gnupg;
```

Comment:
I think this is the result of me copy-pasting and forgetting to change the name.
Can you fix the name?